### PR TITLE
add header and callback function

### PR DESCRIPTION
**Pull request by Yan5577**

### CODE REVIEW

**Comment by ijwilliamson** [comment]
Yan, we need to have a chat about this first thing this morning and figure out what you have done.  There are several commits and we need to be sure they still fit within the project.

File: src/App.js [hunk 1]
```diff
@@ -3,49 +3,56 @@ import Search from "./components/Search";
...
+        <img className="searchlogo" src={SearchIcon} alt="sl"></img>
+
+        <img
+          className="cartlogo"
+          onClick={toggleBasket}
+          src={CartIcon}
+          alt="cart"
+        ></img>
       </div>
+      <Basket
```
Comment (ijwilliamson) [inline src/App.js]: Why did you move the Basket to the end of the html rather than the beginning?

Comment (Yan5577) [reply src/App.js]: this is an old one, please check the latest one, thanks

**Comment by ijwilliamson** [comment]
Closed for a new request

**Review by ijwilliamson — APPROVED** [review]
